### DISCUSSION TITLE
Is there a better way to bundle securely？

### DISCUSSION THREAD

**Discussion by anyfig** [photonstorm/phaser#6520] [Q&A]
i use vite/rollup `npm run build`  to Builds code bundle with production settings (minification, no source maps, etc..) , 
and get a dist folder with  *.js *.html etc.. in it .
looking into these js files , i think it is not too difficult to decode .

Is there a better way to bundle securely？
i want to deploy my game in public server , everyone can play it

**Comment by 201flaviosilva** [comment]
Hey, you mean something like this: https://github.com/javascript-obfuscator/javascript-obfuscator ? :)

**Comment by finefin** [comment]
I am using Parcel https://github.com/parcel-bundler
